ansible-tests/tripleo-ansible-inventory.py

```diff
@@ -211,6 +211,16 @@ def nclient(self):
                 sys.exit(1)
         return self._nclient
 
+    def get_stack_name(self):
+        try:
+            for stack in self.hclient.stacks.list():
+                if 'CREATE' in stack.stack_status:
```

Comment:
My concern here is that this code matches stacks in CREATE_FAILED and CREATE_IN_PROGRESS states. I believe we should restrict stacks to CREATE_COMPLETE status.